include/JSystem/JKernel/JKRAram.h

```diff
@@ -22,13 +24,15 @@ class JKRAramHeap;
 class JKRDecompCommand;
 class JKRAMCommand;
 
-class JKRAramBlock {
+class JKRAramBlock
+{
+public:
   JKRAramBlock(u32 address, u32 size, u32 freeSize, u8 groupID, bool tempMemory);
 
   virtual ~JKRAramBlock();
 
-  JKRAramBlock* allocHead(u32 size, u8 groupID, JKRAramHeap* heap);
-  JKRAramBlock* allocTail(u32 size, u8 groupID, JKRAramHeap* heap);
+  JKRAramBlock *allocHead(u32 size, u8 groupID, JKRAramHeap *heap);
```

Comment:
Can you change the pointer style to the type instead of the variable? 🥺 

Thx

Reply:
Sure, i'll change it, however maybe add a contributors guide to the repo so others don't make the same mistake.
Maybe a script could be made for this? Since now not every file has the same formatting anyway